### ISSUE TITLE
Specify shell version for shell plugin

### ISSUE THREAD

**Issue by bixel** [anishathalye/dotbot#97]
I use dovecot on different OS and ran into a tiny problem trying to expand curly braces like

``` yaml
[shell]
    - ["mkdir -p test/{foo,bar}", Create some directories]
```

This works on e.g. macOS but not on Ubuntu due to the differences in the `subprocess.call`'s default `sh` versions. What do you think of either using the current `$SHELL` as executable, or allowing to define a specific shell for a given command?

I would be happy to implement this if you think it's useful also for other dotbot users.

**Comment by anishathalye** [comment]
Using the current `$SHELL` seems reasonable.

A pull request would be appreciated!

**Comment by anishathalye** [comment]
Fixed in #100.
